src/main/resources/db/demo/kdt/99_verify.sql

```
@@ -206,3 +206,38 @@ FROM `text` WHERE txt_id BETWEEN 8001 AND 8144
+  JOIN approval_revision r ON r.approval_id=ap.approval_id AND r.revision_no=ap.current_revision_no
+  JOIN approval_line l ON l.approval_revision_id=r.approval_revision_id AND l.user_id=ap.user_id
+  WHERE ap.approval_id BETWEEN 8044 AND 8056
+UNION ALL SELECT 'KB ACTIVE없는 IN_PROGRESS결재', COUNT(*) FROM approval ap
+  JOIN approval_revision r ON r.approval_id=ap.approval_id AND r.revision_no=ap.current_revision_no
+  WHERE ap.approval_id BETWEEN 8044 AND 8056 AND ap.status='IN_PROGRESS'
+    AND NOT EXISTS(SELECT 1 FROM approval_line l WHERE l.approval_revision_id=r.approval_revision_id AND l.status='ACTIVE')
+UNION ALL SELECT 'KB activity resource_name 빈값', COUNT(*) FROM activity_log
+  WHERE activity_log_id BETWEEN 8035 AND 8061 AND (resource_name IS NULL OR resource_name='')
+UNION ALL SELECT 'KB 활성 FILE블록 연결없음(껍데기 8394 제외)', COUNT(*) FROM block b JOIN step s ON s.step_id=b.step_id
+  WHERE b.type='FILE' AND b.step_id BETWEEN 8051 AND 8067 AND b.deleted_at IS NULL AND s.status<>'NOT_STARTED'
+    AND NOT EXISTS(SELECT 1 FROM block_file bf WHERE bf.block_id=b.block_id);
```

SQLFluff (4.2.2)

[error] The 'WHERE' keyword should always start a new line.

(LT14)